client/src/ViperProtocol.ts

```diff
@@ -47,6 +52,7 @@ export class Commands {
     static BackendReady = "BackendReady";//BackendReadyParams

```

Comment:
What do these comments mean? e.g. ```//BackendReadyParams```

Reply:
I guess the author wrote them to indicate the type of object the corresponding method takes as input.